scrnaseq.Rmd

```diff
@@ -117,7 +197,8 @@ param$mart_attributes = c(param$annot_main,
                             "percentage_gene_gc_content", "gene_biotype", "strand", "description"))
 param$biomart_mirror = NULL
 
-# Alternatively, we can read a previously compiled annotation table from file
+# Where to store the annotation so that it has to be fetched only one
+# if NULL will be in output_directory/annotation
```

Comment:
if NULL, the annotation will be stored in ...

Reply:
done